### ISSUE TITLE
Failed to resolve: net.time4j:time4j-android:4.0-2018e

### ISSUE THREAD

**Issue by mojtaba-yekta** [MenoData/Time4J#829]
Hi, according to documentation for using Time4A in android I added this line in app gradle:

`implementation group: 'net.time4j', name: 'time4j-android', version: '4.0-2018e'`

but I got this error while syncing gradle:

`Failed to resolve: net.time4j:time4j-android:4.0-2018e`

in mvnrepository.com website the latest version of Time4J Android is 3.44.4-2018e . so I change implementation to this and gradle built successfully:

`implementation group: 'net.time4j', name: 'time4j-android', version: '3.44.4-2018e'`

I just wanted to inform you for this issue

**Comment by MenoData** [comment]
Yes, the readme-file is a little bit too new because v4.0 is not yet released. But I am working intensively on this new version which will also use the tzdb-version 2018f (instead of 2018e).

**Comment by mojtaba-yekta** [comment]
that's great! thanks a lot
